thisReadMe.txt

```diff
@@ -0,0 +1 @@
+And this is going to add us at least one more branch, one more commit, one more file, one more PR etc
```

Comment:
# This is something new.
We were thinking to try out different styling.
So this could be:
- **Bold text**
- _Italic text_
- **_Both text_**
Beside this we can have:

1. Numbered data
2. No clue why I have to type all by myself

- [ #11 #12 ] Task list, something new

@Robert-TK I cannot quote myself...

> Lorem ipsum
> Lorem ipsum 2
> Lorem ipsum 3